src/config.ts

```diff
@@ -396,8 +396,8 @@ export class ConfigurationReader implements vscode.Disposable {
         return ctestJobs;
     }
 
-    get mingwSearchDirs(): string[] {
-        return this.configData.mingwSearchDirs;
+    get additionalCompilerSearchDirs(): string[] {
+        return this.configData.additionalCompilerSearchDirs;
```

Comment:
For backwards compatibility, continue returning `this.configData.mingwSearchDirs` here if `this.configData.additionalCompilerSearchDirs` isn't set (and add `mingwSearchDirs` back to `ExtensionConfigurationSettings`). We can consider the old setting deprecated and not document it, but I'd like to avoid breaking existing configurations if possible.